scripts/linux/buildEdgelet.sh

```diff
@@ -177,7 +177,7 @@ build_project()
     # build project with cross
     cd "$EDGELET_DIR"
 
-    execute cross build -p "$PROJECT" "$BUILD_CONFIG_OPTION" --target "$TOOLCHAIN"
+    execute cross build -p "$PROJECT" --manifest-path=iotedged/Cargo.toml --no-default-features --features runtime-kubernetes "$BUILD_CONFIG_OPTION" --target "$TOOLCHAIN"
```

Comment:
Maybe rename this script to have `kube(rnetes)` in the name so that it doesn't look like it's used for the regular builds.

Reply:
Actually I discovered this is also used to build the `iotedge-diagnostics` binary for the regular `azureiotedge-diagnostics` image, so I guess it's fine as it is.